src/wazuh_modules/syscollector/syscollector_bsd.c

```diff
@@ -27,7 +27,18 @@
 #include <string.h>
 #include <net/route.h>
 
+
 #ifdef __MACH__
+
+#include <stdlib.h>
+#include <stdio.h>
```

Comment:
`stdlib.h` and `stdio.h` are not needed to compile the agent. They are included in other Wazuh headers.